doc/cephfs/mds-journaling.rst

```diff
@@ -0,0 +1,59 @@
+.. _mds-journaling:
+
+====================
```

Comment:
The header isn't quite right, it becomes the title for the page: https://docs.ceph.com/ceph-prs/30396/cephfs/mds-journaling/#cephfs-mds-journaling

Title should be "MDS Journaling"?

Reply:
ah! yes -- fixed